### PR TITLE
add 2.0 in README

### PR DESCRIPTION
**Pull request by Amber1990Zhang**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,12 +30,42 @@ The general **Nebula Graph** documentation structure is as follows:
...
+
+├── Deployment and installation
+    ...
+    ├── Compile and install Nebula Graph
+        ├── ...
+    ├── ...
+...
+```
+
+For detail TOC (table of content), see the following links.
```
Comment (randomJoe211) [inline README.md:63]: For the table of contents, see xxx
detail cannot be used as an adverb.

File: README.md [hunk 2]
```diff
@@ -30,12 +30,42 @@ The general **Nebula Graph** documentation structure is as follows:
...
 
 - [Documentation Index](docs/manual-EN/README.md)
 - [中文文档](https://docs.nebula-graph.com.cn/)
 - [Official Documentation Site](https://docs.nebula-graph.io/)
```
Comment (randomJoe211) [inline README.md:67]: Tag the links as 1.0 docs? And tag 2.0 when we have links for 2.0 docs?

**Review by randomJoe211 — APPROVED** [review]

File: docs-2.0/README.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+* Functions and expressions
+  * [Math](docs-2.0/3.ngql-guide/6.funtions-and-expressions/1.math.md)
+  * [String](docs-2.0/3.ngql-guide/6.funtions-and-expressions/2.string.md)
+  * [Date and time](docs-2.0/3.ngql-guide/6.funtions-and-expressions/3.date-and-time.md)
+  * [Schema](docs-2.0/3.ngql-guide/6.funtions-and-expressions/4.schema.md)
+  * [Case expressions](docs-2.0/3.ngql-guide/6.funtions-and-expressions/5.case-expressions.md)
+
+* General query statements
+  * [Math](docs-2.0/3.ngql-guide/7.general-query-statements/2.match.md)
```
Comment (whitewum) [inline docs-2.0/README.md:45]: Match

**Review by whitewum — APPROVED** [review]

**Review by randomJoe211 — APPROVED** [review]
AWE
SOME
🥇

**Pull request merged**